apps/api/src/app/environments/environment.ts

```diff
@@ -73,4 +73,9 @@ export default {
     username: process.env.SYSLUMENN_USERNAME ?? '',
     password: process.env.SYSLUMENN_PASSWORD ?? '',
   },
+  rskDomain: {
+    username: 'rf_api_island.is',
```

Comment:
No variable needed for the username?

Reply:
This user is already being exposed in plaintext in other parts of the project so opted for setting it here to reduce variable count in dev environment.